src/mumble/API.h

```diff
@@ -91,14 +91,14 @@ public slots:
 								mumble_channelid_t channelID, const char **name,
 								std::shared_ptr< api_promise_t > promise);
 	void getAllUsers_v_1_0_x(mumble_plugin_id_t callerID, mumble_connection_t connection, mumble_userid_t **users,
-							 size_t *userCount, std::shared_ptr< api_promise_t > promise);
+							 std::size_t *userCount, std::shared_ptr< api_promise_t > promise);
```

Comment:
This change is unrelated to your PR, no? If so, please remove it (and others like it).
Or put them in a separate commit - that'd also be fine.

Reply:
This is #6329

@mryamac If you rebase the branch on the current master (after the merge of #6329) it should be fine

Reply:
While squashing commits, I made a mistake, including the last master commit. That's why some std headers still exist. I will create a new PR and replace the new API functions in that PR.

Reply:
As I said, a ``git rebase master`` should fix this (assuming your local master on your computer is up to date)
If you want to try to improve your git skills, you could try that :)
You can also always look up git help on the internet if you have trouble. Most operations/situations are fixable with git